test/parse.jl

```diff
@@ -254,3 +254,6 @@ end
     end
     @test_throws ArgumentError parse(Complex{Int}, "3 + 4.2im")
 end
+
+# added ⟂ to operator precedence (#24404)
+@test Base.operator_precedence(:∥) == Base.operator_precedence(:⟂) == Base.operator_precedence(:<)
```

Comment:
Directly testing what the `operator_precedence` function returns seems kind of pointless since that's what you defined it to be. Testing the result of a few parses would seem to be a better end-to-end test.

Reply:
Something like
```jl
@test Meta.parse("a ⟂ b ⟂ c") == Expr(:comparison, :a, :⟂, :b, :⟂, :c)
@test Meta.parse("a ⟂ b ∥ c") == Expr(:comparison, :a, :⟂, :b, :∥, :c)
```
?